src/test/java/com/wikia/webdriver/common/core/EmailUtils.java

```diff
@@ -64,7 +57,7 @@ public static String getFirstEmailContent(String userName, String password, Stri
           }
         }
         
-        if (i > 15) {
+        if (i > 30) {
```

Comment:
![dirty](http://www.picslyrics.net/images/638018-marilyn-manson-today-i-am-dirty.jpg)

Reply:
what's an 'i'? Try to use more explicit variable names.

Reply:
Good point, I just updated the param value... But yeah, scout rule. ;)

Reply:
I have to admit that question what's an "I" is one of the most magnificent puzzles of all time. Can't give you a shot answer, but maybe on of multiple philosophers of ages, would be able to satisfy you with an answer. :trollface: